tree/dataframe/test/dataframe_merge_results.cxx

```diff
@@ -0,0 +1,291 @@
+#include <ROOT/RDataFrame.hxx>
```

Comment:
You did a very nice coverage here already, but should we have a test for every possible action that can be invoked on RDF? E.g. `Profile{1D,2D}` is not tested.

Reply:
Well yes, I am always in favor of testing more! I would also like to add some test with some "reasonably complicated" RDF graph

Reply:
Or make it a tutorial even

Reply:
if you make a tutorial (which is independent of adding the tests) make sure to make it clear that this is an advanced feature, for developers

Reply:
I've added tests for Profile1D,2D in the C++ tests. For the python tests maybe it's better to stick with a very short number since they take longer to process. For now there are two tests, one for the Mean operation and the other for Histo1D, that also `pickle` the results a couple of times to make sure they can be serialized properly